TDP-4249 add info about length restrictions on recordings for create … (#404)
Bandwidth/bandwidth.github.io: voice/bxml/verbs/record.md

| recordCompleteMethod         | (optional) The HTTP method to use for the request to `recordCompleteUrl`. GET or POST. Default value is POST.                                                                                                          |
| recordingAvailableUrl        | (optional) URL to send the [Recording Available](../callbacks/recordingAvailable.md) event to once it has been processed. Does not accept BXML.                                                                        |
| recordingAvailableMethod     | (optional) The HTTP method to use for the request to `recordingAvailableUrl`. GET or POST. Default value is POST.                                                                                                      |

> | transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Transcription can succeed only for recordings of length greater than 500 milliseconds and less than 4 hours. Default is `false`.                                                                                                                       |

Reviewer: I would say `can only be performed for recordings`. As written it kind of implies failure, whereas it's really just a constraint.

Author: well, you can try a transcription on any recording and it will fail if less than or greater than these constraints. we don't block transcription, we start the process in any situation and it either fails or succeeds. so i think my wording is accurate. do you still disagree?

Reviewer 2: Oh we don't do checks on the recording if you request a transcription via the API? Seems like we should... :thinking: 
Don't we at least check before we send it to our transcription service?

Author: yeah, i followed what was currently in place for the "too long" recordings, where the error comes as a callback, not as an immediate validation error.

Author: so it goes to VMS and comes back, it does not go to the actual transcription service

> | transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Transcription can succeed only for recordings of length greater than 500 milliseconds and less than 4 hours. Default is `false`.                                                                                                                       |

Reviewer 2: Don't need to change the words, but can you change the number of spaces at the end so the table lines up? Doesn't matter to customers, but it's easier to read for us.
If you do change the words though, make sure to change them in each of these files.
```suggestion
| transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Transcription can succeed only for recordings that are longer than 500 milliseconds and less than 4 hours. Default is `false`.          |
```

Reviewer 2: Also `silenceTimeout` has 2 extra spaces on its line (though you didn't add them, might as well remove them).
